tests/structure/test_prototype.py

```diff
@@ -0,0 +1,100 @@
+from typing import Final
+
+import pytest
+from pymatgen.core.structure import Lattice, Structure
+
+from matbench_discovery import TEST_FILES
+from matbench_discovery.structure import prototype
+
+NaCl = Structure(
+    lattice=[[2, 2, 0], [0, 2, 2], [2, 0, 2]],
+    species=["Na", "Cl"],
+    coords=[[0, 0, 0], [0.5, 0.5, 0.5]],
+)
+CsCl = Structure(
+    lattice=[[4, 0, 0], [0, 4, 0], [0, 0, 4]],
+    species=["Cs", "Cl"],
+    coords=[[0, 0, 0], [0.5, 0.5, 0.5]],
+)
+ZnO_zincblende = Structure(
+    lattice=[[2, 2, 0], [0, 2, 2], [2, 0, 2]],
+    species=["Zn", "O"],
+    coords=[[0, 0, 0], [0.25, 0.25, 0.25]],
+)
+ZnO_wurtzite = Structure(
+    lattice=Lattice.from_parameters(
+        a=3.8227, b=3.8227, c=6.2607, alpha=90, beta=90, gamma=120
+    ),
+    species=["Zn", "O", "Zn", "O"],
+    coords=[
+        [1 / 3, 2 / 3, 0],
+        [2 / 3, 1 / 3, 0.3748],
+        [2 / 3, 1 / 3, 1 / 2],
+        [1 / 3, 2 / 3, 1 / 2 + 0.3748],
+    ],
+)
+CaF2_fluorite = Structure(
+    lattice=[[5.46, 0, 0], [0, 5.46, 0], [0, 0, 5.46]],
+    species=["Ca", "F", "F"],
+    coords=[[0, 0, 0], [0.25, 0.25, 0.25], [0.75, 0.75, 0.75]],
+)
+Cu3Au = Structure(
+    lattice=[[3.75, 0, 0], [0, 3.75, 0], [0, 0, 3.75]],
+    species=["Au", "Cu", "Cu", "Cu"],
+    coords=[[0, 0, 0], [0.5, 0.5, 0], [0.5, 0, 0.5], [0, 0.5, 0.5]],
+)
+Fe3Al_DO3 = Structure(
+    lattice=[[5.76, 0, 0], [0, 5.76, 0], [0, 0, 5.76]],
+    species=["Al", "Fe", "Fe", "Fe", "Al", "Fe", "Fe", "Fe"],
+    coords=[
+        [0, 0, 0],
+        [0.25, 0.25, 0.25],
+        [0.5, 0.5, 0],
+        [0.75, 0.75, 0.25],
+        [0, 0.5, 0.5],
+        [0.25, 0.75, 0.75],
+        [0.5, 0, 0.5],
+        [0.75, 0.25, 0.75],
+    ],
+)
+SrTiO3_perovskite = Structure(
+    lattice=[[3.9, 0, 0], [0, 3.9, 0], [0, 0, 3.9]],
+    species=["Sr", "Ti", "O", "O", "O"],
+    coords=[[0, 0, 0], [0.5, 0.5, 0.5], [0.5, 0.5, 0], [0.5, 0, 0.5], [0, 0.5, 0.5]],
+)
+esseneite = Structure.from_file(f"{TEST_FILES}/structures/esseneite.cif.gz")
+
+# TODO figure out who is right, moyopy or spglib, in cases where they differ (marked
+# below, tracked in https://github.com/CompRhys/aviary/pull/96)
+TEST_CASES: Final[tuple[tuple[Structure, str], ...]] = (
+    (NaCl, "AB_cF8_225_a_b:Cl-Na"),
+    (CsCl, "AB_cP2_221_a_b:Cl-Cs"),
+    (ZnO_zincblende, "AB_cF8_216_a_c:O-Zn"),
+    (
+        ZnO_wurtzite,
+        "AB_hP4_186_2b_2b:O-Zn",  # prev with spglib: AB_hP4_186_b_b:O-Zn
```

Comment:
this 2b_2b is still a bug somewhere in moyo implementation as it's incompatible with the pearson symbol (4 at the end means there are only 4 atoms in conventional unit cell but the 2b_2b implies there would be 8). 2 is the multiplicity of the b wp and so maybe theres a dictionary or something in moyo returning 2b instead of just b.

Reply:
maybe addressed by https://github.com/spglib/moyo/pull/64 @lan496?

Reply:
https://github.com/CompRhys/aviary/pull/96#issuecomment-2628696647 seemingly a sorting/groupby issue with the orbits